From 51ba88010de9e0898b4552b88f464caaf143895d Mon Sep 17 00:00:00 2001
From: Michael Pauzner <mpauzner@habana.ai>
Date: Tue, 06 May 2025 13:40:03 +0300
Subject: [PATCH] [SW-221409] gaudi3: add serdes backup load addr

Loading of backup serdes partition image to die1 ARC,
requires definition of the loading address.

Change-Id: If85f774cd266582acfe3c17bf7c62f87125e9a80
Signed-off-by: Michael Pauzner <mpauzner@habana.ai>
---

diff --git a/gaudi3/gaudi3_mem_addr_dt.h b/gaudi3/gaudi3_mem_addr_dt.h
index e8b2a20..db780b8 100644
--- a/gaudi3/gaudi3_mem_addr_dt.h
+++ b/gaudi3/gaudi3_mem_addr_dt.h
@@ -327,4 +327,9 @@
 #define ARC_PID_RSVD_DRAM_TRACE_OFFSET			0x800000
 #define ARC_PID_RSVD_DRAM_SIZE				0x1000000
 
+/* SerDes backup in Die1 Arc1 DCCM */
+#define D1_ARC1_DCCM_HI					0x0300007F
+#define D1_ARC1_DCCM_LO					0xDB080000
+#define D1_ARC1_DCCM_SIZE				0x40
+
 #endif
